assets/stylesheets/layout/_sidebar.scss

```diff
@@ -126,9 +126,7 @@
 		overflow: hidden;
 		display: flex;
 
-		&:focus {
-			outline: none;
-		}
+		-webkit-tap-highlight-color: rgba($gray-dark, .2);
```

Comment:
Is this supported on most browsers? We run our CSS through PostCSS/Autoprefixer which removes the need for the prefix, but I couldn't find this one on caniuse.com so I'm not sure that PostCSS handles it yet.